src/main/java/uk/gov/hmcts/reform/sscs/service/RoboticsService.java

```diff
@@ -223,16 +225,18 @@ private JSONObject createRobotics(RoboticsWrapper appeal) {
         return roboticsAppeal;
     }
 
-    private void sendJsonByEmail(long caseId, Appeal appeal, JSONObject json, byte[] pdf, Map<SscsDocument, byte[]> additionalEvidence, boolean isScottish, boolean isPipAeTo) {
+    private void sendJsonByEmail(long caseId, Appeal appeal, JSONObject json, byte[] pdf, Map<SscsDocument, byte[]> additionalEvidence, boolean isScottish, boolean isPipAeTo, boolean isDigitalCase) {
         Appellant appellant = appeal.getAppellant();
 
         String appellantUniqueId = emailHelper.generateUniqueEmailId(appellant);
 
         log.info("Add robotics default attachments for case id {}", caseId);
         List<EmailAttachment> attachments = addDefaultAttachment(json, pdf, appellantUniqueId);
 
-        log.info("Add robotics additional evidence for case id {}", caseId);
-        addAdditionalEvidenceAttachments(additionalEvidence, attachments);
+        if (!isDigitalCase) {
+            log.info("Add robotics additional evidence for digital case {} and case id {}", isDigitalCase, caseId);
```

Comment:
Shouldn't the log message read: `Add robotics additional evidence for non digital case`?

Reply:
Ok will do the change, I thought isDigitalCase flag will display false so message would be 'digital case false'